tests/util/test_deleteQueue.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+
+from mock import Mock
+
+from codalib import util
+
+
+def test_is_successful(monkeypatch):
+    """
+    Check that deleteQueue does not raise an exception and calls
+    doWebRequest and the response.getcode are called once.
```

Comment:
This docstring seems awkward to me, though I can't quite pin where the problem is. Perhaps changing it to the following would work:
```Check that deleteQueue does not raise an exception and that doWebRequest and response.getcode are called once.```